ui/src/session.ts

```diff
@@ -105,7 +105,6 @@ export const unseal = async (passphrase: string): Promise<void> => {
     notification.error(`Could not unlock the session: ${error.message}`);
     return;
   }
-  notification.info("Unsealing the session...");
```

Comment:
@MeBrei is there any reason why the loading screen wouldn't show and we'd need the notification as a "back up"?

Reply:
I can't think of one. We can always add it back in again, when we notice it...